apps/api/views_drf.py

```diff
@@ -474,3 +477,24 @@ def delete(self, request, addon_id, version_id):
         version = self.get_object()
         version.delete()
         return Response(status=204)
+
+
+class GithubView(APIView):
+
+    def post(self, request):
+        if request.META.get('HTTP_X_GITHUB_EVENT') != 'pull_request':
+            # That's ok, we are just going to ignore it, we'll return a 2xx
+            # response so github doesn't report it as an error.
+            return Response({}, status=202)
```

Comment:
I think it would make sense to reverse your `202` and `200` responses. This one is ignored but the status returned is "Accepted" and the other one gets accepted for processing and returns "OK".